client/src/use/useTrackFilters.ts

```diff
@@ -1,9 +1,11 @@
 import {
   ref, computed, Ref, watch,
 } from '@vue/composition-api';
+import { cloneDeep } from 'lodash';
 import Track, { TrackId } from '../track';
 import { updateSubset } from '../utils';
 
+export const DefaultConfidence = 0.1;
```

Comment:
I think the defaults for the individual types should be 0.  If you have a track that has confidence of 0.05 and you load data in and move the confidence slider below it's default 0.10 to 0 the track won't show up unless you go into the advanced context menu and move the slider down for that specific type as well.  You could have detections hidden in the context menu.

Reply:
Yep, you're right.